docs/src/_includes/components/npm_tabs.macro.html

```diff
@@ -0,0 +1,53 @@
+{%- macro npm_tabs(params) -%}
+<div class="c-tabs" data-tabs>
+   <div class="c-tabs__tablist" data-tablist hidden>
+      <button class="c-tabs__tab" data-tab>npm</button>
+      <button class="c-tabs__tab" data-tab>yarn</button>
+      <button class="c-tabs__tab" data-tab>pnpm</button>
+      <button class="c-tabs__tab" data-tab>bun</button>
+   </div>
+    <div class="c-tabs__tabpanel" data-tabpanel>
+        <h3 class="h6 c-tabs__tabpanel__title">npm</h3>
+
+        ```shell
+        {% if params.comment %}
+            # {{ params.comment | safe }}
+            {{ "\n" }}
+        {% endif -%}
+        npm {{ conversions.toNpmCommands[params.command] | safe }}
+        {% for arg in params.args %}
+            {{ conversions.toNpmArgs[arg] | safe }}
+        {% endfor %}
+        {% for package in params.packages %}
+            {{ package | safe }}
+        {% endfor %} ```
```

Comment:
This gets compile to this:
![Screenshot 2024-11-02 174915](https://github.com/user-attachments/assets/b2f90ea4-c11a-42b3-95fa-a4c9989e4d14)

that is because of indentation this code has, but even if we remove the indentation then it will get compiled to this:

![Screenshot 2024-11-02 175006](https://github.com/user-attachments/assets/8baf4171-c270-4d93-913b-ce0d7b14a988)

i think we should revert this change.